web/satellite/src/components/common/VButton.vue

```diff
@@ -11,18 +11,26 @@
         <slot name="icon" />
         <div v-if="isWhiteGreen" class="greenCheck">&#x2713;</div>
         <div v-if="isGreenWhite" class="whiteCheck">&#x2713;</div>
+        <div v-if="hasTrashIcon" class="trashIcon"><TrashIcon /></div>
```

Comment:
better to name classes to follow BEM trash-icon

Reply:
done